docs/pages/developer-guide.md

```diff
@@ -905,7 +905,9 @@ The most basic set of features present in the YAML representation of the `Servic
 
 ### Containers
 
-Each pod runs inside a single container. The `ServiceSpec` specifies the Docker image to run for that container, the virtual network memberships, and the POSIX resource limits for every task that runs inside that container. In the example below, the soft limit for number of open file descriptors for any task in the "hello" pod is set to 1024, and the hard limit to 2048:
+Each pod runs inside a single container. We can specify the `image` that we want to use (for example, a Docker image), the `rlimits`, and the overlay `networks` to join.  The `image` field specifies the Docker image to run for that pod, the image is run in the Mesos Universal Container Runtime. For a container to have its own IP address it needs to join a virtual network, the only supported network at present is the `dcos` overlay network. Finally, `rlimits` allows you to set POSIX resource limits for every task that runs inside that container. 
+
```

Comment:
suggest: 

Each pod runs inside a single container. The `ServiceSpec` specifies the following:

- The `image` field specifies the image you want to use, for example, a Docker image. The image is run in the [DC/OS Universal Container Runtime](https://dcos.io/docs/1.9/deploying-services/containerizers/ucr/).
- The `networks` field specifies the overlay networks to join. For a container to have its own IP address, it must join a virtual network. The only supported network at present is the `dcos` overlay network.
- The `rlimits` field allows you to set POSIX resource limits for every task that runs inside the container.